sandbox-backend/src/main/java/com/exadel/sandbox/team5/service/EmployeeDiscountService.java

```diff
@@ -0,0 +1,6 @@
+package com.exadel.sandbox.team5.service;
+
+import com.exadel.sandbox.team5.entity.EmployeeDiscount;
+
+public interface EmployeeDiscountService extends CRUDService <EmployeeDiscount>{
```

Comment:
_Minor_: Usually there is should no space between class name and generic type `CRUDService <EmployeeDiscount> {` -> `CRUDService<EmployeeDiscount>`